vars/newrelic.groovy

```diff
@@ -0,0 +1,47 @@
+/***********************************
+
+  newrelic DSL
+
+  notifies newrelic of a deployment
+
+  example usages:
+
+  newrelic(
+    apiKey: 'xxxxxxxxxxxxxxxxxxxx',
+    application: 'myApp',
+    user: 'jenkinsUser'
+  )
+
+  withCredentials([string(credentialsId: 'newrelic-api-key', variable: 'APIKEY')]) {
+    wrap([$class: 'BuildUser']) {
+      newrelic(
+        apiKey: "${APIKEY}",
+        application: 'myApp',
+        user: "${BUILD_USER}"
+      )
+    }
+  }
+
+************************************/
+
+import groovy.json.JsonSlurperClassic
+
```

Comment:
Perhaps we should add `@NonCps` here, due possible serialisation problems of `groovy.json.JsonSlurperClassic`